### DISCUSSION TITLE
General supported platform discussion

### DISCUSSION THREAD

**Discussion by sbonar** [libuv/libuv#4035] [General]
Before I get started in learning libuv I am interested in whether it currently supports iOS/iPadOS since this is not in the Support Platform doc?

Are there any limitations as to what process archs are supported?

Are there any limitations to the number of sockets being used or memory?

TIA.  Really excited to get started and learn as I have need for a cross platform solution like this.

**Comment by vtjnash** [comment]
It builds on CI, although it is not an officially supported platform and the tests do not have CI: https://github.com/libuv/libuv/actions/runs/5079231616/jobs/9124710344
